spring-cloud-netflix-core/src/test/java/org/springframework/cloud/netflix/metrics/servo/ServoMetricReaderTests.java

```diff
@@ -36,7 +36,7 @@ public void singleCompositeMonitorYieldsMultipleActuatorMetrics() {
 				metricNames.get(0));
 		assertEquals("metricName(statistic=max,type=GAUGE,unit=MILLISECONDS)",
 				metricNames.get(1));
-		assertEquals("metricName(statistic=min,type=GAUGE,unit=MILLISECONDS)",
+		assertEquals("metricName(statistic=totalOfSquares,type=NORMALIZED,unit=MILLISECONDS)",
```

Comment:
@tbak @jkschneider I'm a little concerned that I had to change this unit test to pass. The min, GAUGE is replaced by totalOfSquares, NORMALIZED. Is that expected?